src/website/app/demos/Card/examples/card/index.js

```diff
@@ -20,4 +20,4 @@ import arbitraryChildren from './arbitraryChildren';
 import clickable from './clickable';
 import orderOfSections from './orderOfSections';
 
-export default [card, arbitraryChildren, clickable, orderOfSections];
+export default [card, orderOfSections, clickable, arbitraryChildren];
```

Comment:
Sorry if I missed this the first time around, but I disagree with this order change. For a few reasons:

1. We're not even sure if freeform order of sections is a thing we want; it's just something we made sure was at least possible as we built it.
2. Even if it is, it's less important than the other examples.
3. We know from experience that we need to draw attention to the use of CardBlock. (Maybe we should even be using a Callout for that note in the #arbitrary-children example?)

Reply:
reverted.

Reply:
the Callout component is having trouble rendering nested markdown in it's children. we could try to fix it or revisit later, up to you.